server/src/main/java/com/objectcomputing/checkins/services/skills/SkillServices.java

```diff
@@ -36,6 +38,12 @@ protected Skill readSkill(UUID skillId) {
 
     }
 
+    public Set<Skill> readAll() {
+        Set<Skill> actionItems = new HashSet<>();
```

Comment:
Please can you update variable name because I think actionItems is for action item rest service ?